src/Microsoft.Dnx.Host.CoreClr/DomainManager.cs

```diff
@@ -11,8 +12,24 @@
 [SecurityCritical]
 sealed class DomainManager
 {
+    [StructLayout(LayoutKind.Sequential)]
+    private unsafe struct NativeBootstrapperContext
+    {
+        public char* OperatingSystem;
+
+        public char* OsVersion;
+
+        public char* Architecture;
+
+        public uint NumProcessors;
```

Comment:
I specifically asked for number of physical processors, which you can't get from .NET (`Environment`). We'd use this by default in Kestrel to set the number of threads to use to match the number of physical cores in the machine.